### DISCUSSION TITLE
available list of kdb Names for Keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by mrossde** [quarto-dev/quarto-cli#9087] [Q&A] [answered]
### Description

Hello everyone!

i was reading the documentation regarding using kdb Keyboard Shortcuts [documented here](https://quarto.org/docs/authoring/markdown-basics.html#keyboard-shortcuts)

Is there a List of available Keyboard Names to use inside that kdb command, do i find it somewhere in the sourcecode maybe if there is no list?

Thanks a lot for Answering and Developing Quarto - its an awesome tool for Building Lectures!!
Regards, Mario

**Comment by mcanouil** [comment, marked as answer]
There are no keywords. Simply standards names of the keys as defined for `kbd` HTML tag.

The shortcode does not check if the key even is a real one, `{{< kbd Blue-Smurf-White-Hat >}}` will work the same.

PS: could you fix your link?

**Reply by mrossde** [reply]
Thanks for the quick response! Link fixed
